airbyte-integrations/connectors/source-pinterest/metadata.yaml

```diff
@@ -19,6 +19,14 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      1.0.0:
+        message:
+          This release updates the date-time fields to use the Airbyte format `timestamp_without_timezone`.
+          This change affects all streams where date-time fields are present, ensuring more accurate and standardized time representations.
+          Users will need to refresh the source schema and reset affected streams after upgrading.
```

Comment:
and reset streams BoardPins, BoardSectionPins, Boards, Catalogs, and CatalogFeeds after upgrading.

Reply:
This release updates the date-time fields to use the Airbyte format `timestamp_without_timezone`.
          This change affects all streams where date-time fields are present, ensuring more accurate and standardized time representations: BoardPins, BoardSectionPins, Boards, Catalogs, and CatalogFeeds.
          Users will need to refresh the source schema and reset affected streams after upgrading.

Reply:
Fixed